packages/datagateway-dataview/src/views/card/isis/isisStudiesCardView.component.tsx

```diff
@@ -0,0 +1,251 @@
+  const dateFilter = (label: string, dataKey: string): React.ReactElement => (
+    <DateColumnFilter
+      label={label}
+      value={filters[dataKey] as DateFilter}
+      onChange={(value: { startDate?: string; endDate?: string } | null) =>
+        pushFilters(dataKey, value ? value : null)
+      }
+    />
+  );
+
+  React.useEffect(() => {
+    fetchIds(parseInt(instrumentId));
+  }, [fetchIds, instrumentId]);
+
+  React.useEffect(() => {
+    if (!fetchedCount) {
+      fetchCount(allIds);
+      setFetchedCount(true);
+    }
+  }, [fetchedCount, allIds, fetchCount, setFetchedCount]);
+
+  const pathRoot = 'browseStudyHierarchy';
+  const instrumentChild = 'study';
+
+  return (
+    <CardView
+      data={data}
+      totalDataCount={totalDataCount}
+      loadData={(params) => fetchData(allIds, params)}
+      loadCount={() => fetchCount(allIds)}
+      loading={loading}
+      sort={sort}
+      filters={filters}
+      query={query}
+      onPageChange={pushPage}
+      onResultsChange={pushResults}
+      onSort={pushSort}
+      onFilter={pushFilters}
+      clearData={clearData}
+      title={{
+        label: t('studies.name'),
+        dataKey: 'STUDY.NAME',
+        content: (studyInvestigation: StudyInvestigation) =>
+          tableLink(
+            `/${pathRoot}/instrument/${instrumentId}/${instrumentChild}/${studyInvestigation.STUDY.ID}/investigation`,
+            studyInvestigation.STUDY.NAME,
+            view
+          ),
+        filterComponent: textFilter,
+      }}
+      description={{
+        label: t('facilitycycles.description'),
+        dataKey: 'STUDY.DESCRIPTION',
+        filterComponent: textFilter,
+      }}
+      information={[
+        {
+          icon: <Fingerprint />,
+          label: t('studies.rb_number'),
+          dataKey: 'INVESTIGATION.VISIT_ID',
+          filterComponent: textFilter,
+        },
+        {
+          icon: <CalendarToday />,
+          label: t('facilitycycles.start_date'),
+          dataKey: 'STUDY.STARTDATE',
+          filterComponent: dateFilter,
+        },
+        {
+          icon: <CalendarToday />,
+          label: t('facilitycycles.end_date'),
+          dataKey: 'STUDY.ENDDATE',
+          filterComponent: dateFilter,
+        },
+      ]}
+    />
+  );
+};
+
+const mapDispatchToProps = (
+  dispatch: ThunkDispatch<StateType, null, AnyAction>
+): ISISStudiesCVDispatchProps => ({
+  fetchIds: (instrumentId: number) =>
+    dispatch(
+      fetchAllIds('investigation', [
+        {
+          filterType: 'where',
+          filterValue: JSON.stringify({
+            'INVESTIGATIONINSTRUMENT.INSTRUMENT.ID': { eq: instrumentId },
+          }),
+        },
+      ])
+    ),
+  fetchData: (allIds: number[], offsetParams: IndexRange) =>
+    dispatch(
+      fetchStudies({
+        offsetParams,
+        additionalFilters: [
+          {
+            filterType: 'where',
+            filterValue: JSON.stringify({
+              INVESTIGATION_ID: { in: allIds },
```

Comment:
This is a clever way to add instrument context to a study search! Again, unsure if this is going to be the final way we fetch study info but it's fine for now